package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/angular/angular-cli.git"
   },
   "engines": {
-    "node": ">= 4.1.0"
+    "node": ">= 5.0.0"
```

Comment:
I don't think we want to exclude anyone still on node 4 even though were not running tests for it.